packages/ui/src/theme/tokens/components/switchField.ts

```diff
@@ -72,6 +81,18 @@ export interface SwitchFieldTokens {
   track: SwitchFieldTrackTokens;
 }
 
```

Comment:
Do we want to comment out all the places we would like to remove after next major release?